demos/classification_demo/main.cpp

```diff
@@ -233,12 +233,13 @@ int main(int argc, char *argv[]) {
 
         // --------------------------------------Configure model output---------------------------------------
         auto outputInfo = network.getOutputsInfo();
-        if (outputInfo.size() != 1) {
-            throw std::logic_error("The network should have only one output.");
+        if (outputInfo.size() < 1 || outputInfo.size() > 3) {
```

Comment:
```suggestion
        if (outputInfo.size() > 3) {
```
That's strange to expect that a net doesn't have any output.